### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'datasets/11/train/'

### ISSUE THREAD

**Issue by hhf-hd** [coordinated-systems-lab/VesselIntentModeling#1]
hello ,thanks for you great work !
when i reproduce yourwork,i get a error like this .
and i wonder if there is something mistake？
![image](https://github.com/coordinated-systems-lab/VesselIntentModeling/assets/26891442/4ec248d0-f163-456d-962c-211802030661)

**Comment by burg54** [comment]
Did you download the data in step 1 and pre-process the data in step 2?

**Comment by hhf-hd** [comment]
yes,I deal with it step by step，but it did not create folder train .
what's more ,when i create the train folder by myself ,i meet the problem as the picture show:
![image](https://github.com/coordinated-systems-lab/VesselIntentModeling/assets/26891442/f61df48d-01fd-4f0a-9de2-6b32647f1cea)

i think the following step did not create any .pt file ,
can you tell me what's wrong with it?

**Comment by burg54** [comment]
Thank you for pushing on this. I should have mentioned last week but I forgot: we are going to refactor the code and push it, which should fix this bug as well as a few others we're aware of. Stay tuned...

**Comment by kbhcyj** [comment]
> 안녕하세요, 훌륭한 작업에 감사드립니다! 제가 당신의 작업을 재현하려고 하면 이런 오류가 발생합니다. 뭔가 실수가 있는 걸까요? 
![스크린샷 2025-01-03 153236](https://github.com/user-attachments/assets/bf9139bc-4334-4c31-b929-c7508dd71320)

The same problem occurs to me too.
Have you solved this issue?

**Comment by burg54** [comment]
I believe there is an issue in `train_utils.py` and `data.py`, which both do some things with strings and path names.

The `get_dirs()` method in `train_utils.py` is hard-coded to look for and/or make a directory called ./data/<>

While the `load_data()` method in `data.py` looks for ./datasets/<>

Have you tried modifying the load_data method? (or the get_dirs method)

**Comment by burg54** [comment]
@kbhcyj  and @hhf-hd everyone, I created a branch called updated_paths that cleans this up, among a few other things. The README is correspondingly slightly updated.

@hhf-hd I sincerely apologize it took so long, I totally forgot about this.

**Comment by burg54** [comment]
I cleaned up a few other things and also add a yaml file for installing dependencies. Again, this is all on the `updated_paths` branch.

**Comment by burg54** [comment]
This update has now been merged into `master` and I believe this issue should be closed.
